drop-transfer/src/service.rs

```diff
@@ -54,6 +55,7 @@ impl Service {
         config: Arc<DropConfig>,
         moose: Arc<dyn Moose>,
         auth: Arc<auth::Context>,
+        init_time: Option<Instant>,
```

Comment:
Is `init_time` needed? I think we can always set `init_time` to `Instant::now()` inside this function. I know there are a couple of steps before this function is called but these things are not doing any network IO or something so the delay is pretty negligible

Reply:
yeah, in this case it is needed, during the talk where we decided to introduce this metric, the two main points were because of the database opening and moose, which are the two main things that happen before going into service

Reply:
Ok :+1:

Reply:
I've checked the code and it seems that the `init_time` can be `Instant` (without option). The `None` value is used in udrop only but since it's a demo, there's no need to optimize the API for it and we can simply use `Instant::now()` in the udrop

Reply:
Good point, done